src/main/java/io/jenkins/pluginhealth/scoring/http/ProbesController.java

```diff
@@ -59,6 +63,30 @@ public ModelAndView list() {
         return modelAndView;
     }
 
+    @GetMapping(path = "/results")
+    public ModelAndView listProbeResults() {
+        final ModelAndView modelAndView = new ModelAndView("probes/results");
+
+        modelAndView.addObject(
+            "probeResults",
+            getProbeResultsData()
+        ).addObject(
+            "pluginsCount",
+            pluginService.getPluginsCount()
+        );
+
+        return modelAndView;
+    }
+
+    public Map<String, Long> getProbeResultsData() {
+        Map<String, Long> probeResultsMap = probes.stream()
+            .collect(Collectors.toMap(Probe::key, probe -> pluginService.getProbeRawData(probe.key())));
+
+        probeResultsMap.remove("last-commit-date");
+
+        return probeResultsMap;
+    }
+
```

Comment:
Normally, we don't mix view layer with service layer. 
I'll be more comfortable having this method in a `ProbeService` which would be injected in this controller.

Reply:
Done. I also moved  `getProbesRawResultsFromDatabase` (previously called `getProbeRawData`) from `PluginService` to `ProbeService`. And changed the method names to try to make it a little clear what they do.